src/Styleguide/Artwork/ImageCarousel.tsx

```diff
@@ -0,0 +1,209 @@
+import React from "react"
+import styled from "styled-components"
+
+import { Flex } from "Styleguide/Elements/Flex"
+import Icon from "Components/Icon"
+import { Responsive } from "../Utils/Responsive"
+
+/**
+ * The image should be placed 100px from the top of the document and leave 60px
+ * for the buttons below the image.
+ */
+const ButtonsContainerHeight = 60
+const ButtonsContainerWithPageIndicatorsHeight = ButtonsContainerHeight + 20
+const ImageContainerViewportMargin = 100 + ButtonsContainerHeight
+
+const Container = styled(Flex)`
+  /* background-color: #0fdb82; */
+`
+
+const ImageAreaContainer = styled(Flex)`
+  width: 100%;
+  min-height: 450px;
+  height: calc(100vh - ${ImageContainerViewportMargin}px);
+  line-height: 0; /* Don’t introduce visual margins */
+`
+
+const ImageContainer = styled(Flex)`
+  height: 100%;
+  width: 100%;
+`
+
+const Image = styled.img`
+  cursor: zoom-in;
+  max-height: 100%;
+  max-width: 100%;
+`
+
+// TODO: Should Icon have this styling by default?
+const Button = styled.a`
+  ${Icon /* sc-selector */} {
```

Comment:
@alloy - I saw your message the other day but still don't quite understand what this comment does? Is this a magic comment similar to how webpack does it?

Reply:
Might be easier to compare to a magic comment that disables a linter rule.